website/docs/cloud-docs/agents/requirements.mdx

```diff
@@ -0,0 +1,42 @@
+---
+page_title: Terraform Cloud Agents - Requirements - Terraform Cloud and Terraform Enterprise
+description: >-
+  Supported operating systems, Terraform versions, hardware requirements, and networking requirements for Terraform Cloud Agents.
+---
+
+# Requirements
+
```

Comment:
~> **Important:** All agents in an agent pool should have the same operating system and hardware resources available. If the agents are using different resources, run performance can vary significantly between each agent in the pool.